examples/polyglot-zipkin/docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       - "2181:2181"
       - "9092:9092"
     environment:
-      - ADVERTISED_HOST=${TRACING_SERVER}
+      - ADVERTISED_HOST=${KAFKA_ZOOKEEPER}
```

Comment:
No. The problem we run into is probably because of wrong configuration. Zookeeper or kafka were in inconsistent state.